Source/Core/PixelFormat.js

```diff
@@ -144,6 +146,28 @@ define([
         /**
          * @private
          */
+        componentLength : function(pixelFormat) {
```

Comment:
Perhaps `componentSizeInBytes`?  "Length" sounds like, for example, `RGB` would be three: R, G, and B.